src/components/WorkDescription/ProblemSolution.js

```diff
@@ -43,28 +62,41 @@ export default ({ problem, solution }) => {
 
   return (
     <>
-      <Text size="normal" weight={700}>
+      <Text size="normal" color="#561dee" weight={700}>
```

Comment:
Is there a colour variable you could use here?

Reply:
I think because this prop was defined as a string I can't use a variable. I think after the site is live I will go back and replace all colours with variables @rosejbon